NohanaImagePicker/AssetListSelectableDateSectionController.swift

```diff
@@ -32,6 +33,16 @@ class AssetListSelectableDateSectionController: UICollectionViewController, UICo
         let cellWidth = (view.frame.width - cellMargin * (CGFloat(numberOfColumns) - 1)) / CGFloat(numberOfColumns)
         return CGSize(width: cellWidth, height: cellWidth)
     }
+
+    private var isHiddenMenu: Bool {
+        switch PHPhotoLibrary.authorizationStatus() {
```

Comment:
[authorizationStatus](https://developer.apple.com/documentation/photokit/phphotolibrary/1620745-authorizationstatus) function is supported for iOS13.
Please use [authorizationStatus(for:)](https://developer.apple.com/documentation/photokit/phphotolibrary/3616052-authorizationstatus) by over iOS 14

```example.swift
let status: PHAuthorizationStatus
if #available(iOS 14, *) {
    status = PHPhotoLibrary.authorizationStatus(for: .readWrite)
} else {
    status = PHPhotoLibrary.authorizationStatus()
}
switch status { ...
```